sbpy/units/core.py

```diff
@@ -133,3 +136,150 @@ def spectral_density_vega(wfb):
         (flam0.unit, VEGA, lambda x: x / flam0.value,
          lambda x: x * flam0.value)
     ]
+
+
+@u.quantity_input(cross_section='km2', reflectance='1/sr',
+        f_sun=['W/(m2 um)', 'W/(m2 Hz)'])
+def reflectance(cross_section=None, reflectance=None, wfb=None, M_sun=None,
+        f_sun=None):
+    """Reflectance related equivalencies.
+
+    Support conversion from/to reflectance and scattering cross-section
+    to/from total flux and magnitude.
+
+    The default magnitude system is ``VEGAmag``, where the apparent magnitude
+    of Vega is assumed to be 0 in all and any wavelengths and bands.  If other
+    magnitude system is used, then it is implicitly inferred from the solar
+    magnitude passed by keyword 'M_sun'.
+
```

Comment:
It is covered in the `M_sun` documentation, but it may help the user to quote the default values in the summary, and that these are not affected by the sbpy solar spectrum calibration system.  If the user wants something based on the current default solar spectrum, they need to provide `wfb`.

Reply:
I don't understand the part starting from "..., and that these are not affected ..." through the end.

Reply:
Got it.  Thanks.